docs/docs/en/guide/task/pytorch.md

```diff
@@ -0,0 +1,117 @@
+# Pytorch Node (experimental)
+
+## Overview
+
+[Pytorch](https://pytorch.org) is a mainstream Python machine learning library.
+
+In order to enable users to **run Pytorch project in DolphinScheduler more convenient,** Pytorch task plugin is implemented. It provides convenient Python environment management and supports the running of Python projects.
+
+Unlike the Python task plugin, this component allows users to **quickly use existing Python environments or create new ones (using Virtualenv or Conda)**. **Support for running Python projects (native or Git projects)** instead of just Python scripts.
```

Comment:
```suggestion
`Pytorch task plugin` is more than `Python task plugin`, which allows users to **conveniently use existing Python environments or create new ones (using `Virtualenv` or `Conda`)**. **It supports running Python projects (native or Git projects)** instead of just Python scripts.
```